### DISCUSSION TITLE
Welcome to QuickRawPicker Discussions!

### DISCUSSION THREAD

**Discussion by qdwang** [RawLabo/QuickRawPicker#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by xtrul** [comment]
hI, I LOVE THIS, can you please make a list with all the shortcuts, maybe for the future dev, any chance for a telegram chat group? (check your telegram)

**Reply by qdwang** [reply]
All the shortcuts are listed [here](https://github.com/qdwang/QuickRawPicker#shortcut-list). 

Sorry but I'd like to keep telegram for my personal use. 

Alternatively, a Discord channel has already been set up. [https://discord.gg/zTRdVKkbNT](https://discord.gg/zTRdVKkbNT)
